packages/bot-web-ui/src/analytics/__tests__/utils.spec.ts

```diff
@@ -1,15 +1,15 @@
+import { DBOT_TABS } from 'Constants/bot-contents';
 import { TFormStrategy } from '../constants';
-import { getTradeParameterData } from '../utils';
+import {
+    getTradeParameterData,
+    getRsDropdownTextFromLocalStorage,
+    getSubpageName,
+    rudderstack_text_error,
+} from '../utils';
 
-jest.mock('@deriv/bot-skeleton/src/scratch/blockly', () => jest.fn());
 jest.mock('@deriv/bot-skeleton/src/scratch/dbot', () => jest.fn());
-jest.mock('@deriv/bot-skeleton/src/scratch/hooks/block_svg', () => jest.fn());
 
 describe('utils', () => {
-    beforeEach(() => {
-        jest.clearAllMocks();
-    });
-
```

Comment:
Starting with version 27, you don't have to manually call **jest.clearAllMocks()** in each test, as Jest automatically clears all mocks between tests. 
We use `"babel-jest": "^27.3.1"`